server/models/Request.js

```diff
@@ -0,0 +1,35 @@
+const mongoose = require("mongoose");
+
+const requestSchema = new mongoose.Schema({
+    user_id:{
```

Comment:
Let's follow JS casing conventions here and modify these fields to be in `camelCase` 👍🏼

Reply:
Do you mean, instead of using user_id, use userId?

Reply:
Precisely!